.circleci/config.yml

```diff
@@ -489,7 +489,15 @@ workflows:
       - forge_test:
           context:
             - circleci-repo-readonly-authenticated-github-token
-      - template_regression_tests
+      # This is a long running job, so we only run it on the main branch. It's encouraged that task
```

Comment:
This is starting to be ~9 minutes. It's a valuable test suite to have but running it on every PR is slow.